workflows/transcriptomics/goseq/README.md

```diff
@@ -0,0 +1,29 @@
+# Goseq GO-KEGG Enrichment Analysis Workflow
+
+## Inputs dataset
+
+The workflow need the following inputs:
+- The DEG file:
+    - A tabular file with first column the gene symbol and second column a boolean value whether the gene is DEG or not. 
+- The gene length file:
+    - A tabular file with first column the gene symbol and second column the gene length of the genes.
```

Comment:
Could you indicate the potential users where they can get this info? For example using a GTF and the tool "Gene length and GC content from GTF and FASTA file (Galaxy Version 0.1.2)"